ts/features/walletV3/payment/components/WalletPaymentConfirmContent.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import {
+  Body,
+  GradientScrollView,
+  LabelLink,
+  ListItemHeader,
+  ModuleCheckout,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { useNavigation } from "@react-navigation/native";
+import { openAuthenticationSession } from "@pagopa/io-react-native-login-utils";
+import {
+  AppParamsList,
+  IOStackNavigationProp
+} from "../../../../navigation/params/AppParamsList";
+import { WalletPaymentRoutes } from "../navigation/routes";
+import {
+  WALLET_PAYMENT_TERMS_AND_CONDITIONS_URL,
+  getPaymentLogo
+} from "../../common/utils";
+import { format } from "../../../../utils/dates";
+import { capitalize } from "../../../../utils/strings";
+import { PaymentRequestsGetResponse } from "../../../../../definitions/pagopa/ecommerce/PaymentRequestsGetResponse";
+import { Bundle } from "../../../../../definitions/pagopa/ecommerce/Bundle";
+import { WalletInfo } from "../../../../../definitions/pagopa/walletv3/WalletInfo";
+import {
+  TypeEnum,
+  WalletInfoDetails,
+  WalletInfoDetails1,
+  WalletInfoDetails2,
+  WalletInfoDetails3
+} from "../../../../../definitions/pagopa/walletv3/WalletInfoDetails";
+import I18n from "../../../../i18n";
+
+import { formatNumberCurrencyCentsOrDefault } from "../../../idpay/common/utils/strings";
+import { WalletPaymentTotalAmount } from "./WalletPaymentTotalAmount";
+
+export type WalletPaymentConfirmContentProps = {
+  paymentMethodDetails: WalletInfoDetails;
+  selectedMethod: WalletInfo;
+  selectedPsp: Bundle;
+  paymentDetails: PaymentRequestsGetResponse;
+  isLoading?: boolean;
+  onConfirm?: () => void;
+};
+
+export const WalletPaymentConfirmContent = ({
+  paymentMethodDetails,
+  selectedPsp,
+  selectedMethod,
+  paymentDetails,
+  isLoading,
+  onConfirm
+}: WalletPaymentConfirmContentProps) => {
+  const navigation = useNavigation<IOStackNavigationProp<AppParamsList>>();
+
+  const taxFee = selectedPsp.taxPayerFee ?? 0;
+
+  const paymentAmount = paymentDetails.amount ?? 0;
+
+  const totalAmount = +paymentAmount + taxFee;
+
+  return (
+    <GradientScrollView
+      primaryActionProps={{
+        label: `${I18n.t(
+          "payment.confirm.pay"
+        )} ${formatNumberCurrencyCentsOrDefault(totalAmount)}`,
```

Comment:
This function si exported from ID Pay which is another feature. My suggestion is to move the utility function outside the ID Pay feature and transform it in a "common" utility function or use `formatNumberCentsToAmount` function

```suggestion
        )} ${formatNumberCentsToAmount(totalAmount, true, "right")}`,
```

Reply:
Addressed into f6100127dec994cf420a54d5f41aa965a87e972e